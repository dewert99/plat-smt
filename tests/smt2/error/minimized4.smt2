(declare-const c Bool)
(check-sat)
(check-sat-assuming)
