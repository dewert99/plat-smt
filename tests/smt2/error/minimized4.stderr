the function `check-sat-assuming` expects at least 1 arguments but only got 0 at 3:20
(check-sat-assuming)
                   ^

